man/chpl.rst

```diff
@@ -233,11 +233,12 @@ OPTIONS
 
 **\--[no-]warn-param-implicit-numeric-conversions**
 
-    When used in conjunction with ``warn-int-uint``,
-    ``--warn-real-real``, or ``--warn-integral-integral``, this flag
-    enables [or disables] these compilation warnings about implicitly
-    converting between numeric types to also apply when the converted
-    value is a ``param``.
+    When used in conjunction with ``--warn-int-to-uint``,
+    ``--warn-small-integral-to-float``, ``--warn-integral-to-float``,
+    ``--warn-float-to-float``, or ``--warn-integral-to-integral``,
+    this flag enables [or disables] these compilation warnings about
+    implicitly converting between numeric types to also apply when the
+    converted value is a ``param``.
```

Comment:
Yes, this is accurate. The `--warn-implicit-numeric-conversions` also enables these flags specifically named here. So you could list that as well. Perhaps it would be better to write it as "When used in conjunction with any of the above flags enabling compilation warnings for numeric implicit conversions, this flag enables [or disables] these compilation warnings to also apply when the converted value is a ``param``."